docs/sources/http_api/reporting.md

```diff
@@ -0,0 +1,69 @@
++++
+title = "Reporting API"
+description = "Grafana Enterprise APIs"
+keywords = ["grafana", "enterprise", "api", "reporting"]
+aliases = ["/docs/grafana/latest/http_api/reporting/"]
+type = "docs"
+[menu.docs]
+name = "Reporting API"
+parent = "http_api"
++++
+
+# Reporting API
+
+> Reporting is only available in Grafana Enterprise. Read more about [Grafana Enterprise]({{< relref "../enterprise" >}}).
+
+The Reporting API allows for programmatic access to [Reporting]({{< relref "../enterprise/reporting.md" >}}) functionality.
+
+## Send report
```

Comment:
```suggestion
## Send a report
```

Reply:
Whoops, had forgotten to push that fix 😇